service_manifest.yml

```diff
@@ -200,11 +200,9 @@ heuristics:
 
   - heur_id: 12
     name: Visual Basic and JavaScript
-    score: 1
+    score: 500
     filetype: '.*'
     description: Sample uses a combination of both Visual Basic and JavaScript
-    signature_score_map:
-      suspicious_url_found: 500
```

Comment:
Do you want to modify the _look_for_iocs_between_vb_and_js function in jsjaws.py to remove the `url_sec.heuristic.add_signature_id("suspicious_url_found")`?